JoinColony/colonyCDapp
# fix: loading skeleton for loading actions with reputation

## Description

Fix loading skeleton when open sidebar with reputation action.

## Testing
- Create action with reputation eg. Simple payment
- Open action sidebar with reputation
- Loader spinner on the right is no longer appears

## Diffs

**New stuff** ✨

Add information about motion loading to context

**Changes** 🏗

Before:

[loading-wheel.webm](https://github.com/user-attachments/assets/6524de6a-ca86-4353-837e-416ef2673496)

After:

https://github.com/user-attachments/assets/907dc754-f741-4301-914a-4961de988952



Resolves https://github.com/JoinColony/colonyCDapp/issues/3299
